week-03/main.js

```diff
@@ -13,7 +14,13 @@ var list = [1, 2, 3, 4, 5];
 // and `hobbies` an array.
 // Assign the object to the variable `user`
 
+var user = {
+  'name': 'Kryst',
+  'isAlive': true,
+  'hobbies': []
+};
 
+user.hobbies = 'golf';
```

Comment:
This reassigns `hobbies` to a string. How would you add `golf` to the hobbies array?